package ExceptionHnadling;

public class arthimaticExc {

	public static void main(String[] args) {
	 int a= 2;
	 int b=0;
	 
	 
	 try {
		 
		 int c= a/b;
	}
 catch(ArithmeticException d) {
		 System.out.println("i'll exicute First");
	 }
	 catch(Exception d) {
		 System.out.println("GGDGRDRD");
		 
	 }
	 
	 
	}}
	
